yt_dlp/extractor/urplay.py

```diff
@@ -108,22 +108,25 @@ def parse_lang_code(code):
             if code is None:
                 return
             lang = code.lower()
-            if not ISO639Utils.long2short(lang):
-                lang = ISO639Utils.short2long(lang)
+            # bypass for Persian
+            if lang != "per":
```

Comment:
pls explain this

Reply:
Revert this change, and add Persian language to ISO639Utils in next commit